cargo-concordium/src/build.rs

```diff
@@ -721,14 +746,13 @@ pub(crate) fn build_and_run_integration_tests(
     build_options: BuildOptions,
     test_targets: Vec<String>,
 ) -> anyhow::Result<()> {
+    let cargo_args = build_options.cargo_args.clone();
```

Comment:
```suggestion
    let cargo_args = &build_options.cargo_args;
```

Reply:
The function one line `crate::handle_build()` below moves the value, so I had to clone it here.